var/spack/repos/builtin/packages/adlbx/package.py

```diff
@@ -12,12 +12,20 @@ class Adlbx(AutotoolsPackage):
 
     homepage = 'http://swift-lang.org/Swift-T'
     url      = 'http://swift-lang.github.io/swift-t-downloads/spack/adlbx-0.0.0.tar.gz'
+    git      = "https://github.com/swift-lang/swift-t.git"
 
+    version('develop', branch='master')
```

Comment:
Now that #1983 has been merged, you can actually name this version `master` if you want and Spack will handle it correctly. Same for other packages.

Reply:
Addressed in c575882d9 .